env/git.go

```diff
@@ -112,26 +112,21 @@ func GitSHA(path string) (string, error) {
 }
 
 var GitBlob = func(path string, commit *object.Commit) (string, error) {
-	if commit == nil {
-		blob, err := fallbackBlob(path)
+	if commit != nil {
+		if file, err := commit.File(path); err == nil {
```

Comment:
Sorry, I was waiting for another code change.

I took me 5 readings of this to understand `if x, err : = ...; err == nil {`. That's why I didn't think `err` was being checked before, I missed the second clause.